Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs

```diff
@@ -126,6 +127,9 @@ private void Fire(InputAction.CallbackContext ctx)
 
     private void ArrowSelect(InputAction.CallbackContext ctx)
     {
+        if (arrowHeight < lowestCheckHeight)
+            return;
```

Comment:
Nice catch! I started wondering why the arrows were missed before they even show up.

This makes me wonder why we don't have any visual response when we register a _miss_, though (as opposed to the arrow being ignored and going all the way up)

Reply:
The main reason is that players would have stand completly still in order to not always get misses when playing with a gamepad

Reply:
Aha, understandable

Reply:
I do think we could have some sort of detection of when players actively flick the stick, but that sounds too fancy for just a miss indicator